src/pages/index.tsx

```diff
@@ -69,6 +69,14 @@ const ApplicationRouter = (): JSX.Element | null => {
         </Route>
       </Route>
 
+      <Route path="/test">
```

Comment:
Please remove `/` sign from here

Reply:
Remove this part entirely, thank you. Done